content/data-reporting/changelog/index.mdx

```diff
@@ -50,6 +50,7 @@ The developer preview version provides a preview of the upcoming changes to the
     - `filters.surveys.type`
     - `filters.surveys.answer_id`
     - `filters.events.types`
+- All methods requires the `reports_read` scope. This applies to **v3.** versions of the _Reports API_.
```

Comment:
```suggestion
- All methods require the `reports_read` scope. This applies to all **v3.x** versions of the _Reports API_.
```

Reply:
I'd also move this note above the renamed fields list, under this point:
> To provide more consistent experience across all Platform APIs, filters are now aligned with those used in List Archives (the Agent Chat API).

it gets "lost" when it's under a long list like that :D

Reply:
Corrected.